### DISCUSSION TITLE
[3590]植物大战僵尸

### DISCUSSION THREAD

**Discussion by 2336351763** [pjy612/SteamManifestCache#435] [Share]
`appid:3590` 

keys
```
3591;1625fdac425750d0736c420f3877d3b4b4119d1d361629c2d12b360eb3689839
```
[3590_config.vdf.zip](https://github.com/pjy612/SteamManifestCache/files/14093667/3590_config.vdf.zip)
邮箱：2336351763@qq.com

**Comment by pjy612** [comment]
* 请调整密钥格式  #26

**Reply by 2336351763** [reply]
🆗
